radial_kernel.hpp

```diff
@@ -175,12 +177,18 @@ class RadialDispersalKernel
     std::cauchy_distribution<double> cauchy_distribution;
     std::exponential_distribution<double> exponential_distribution;
     von_mises_distribution von_mises;
+    DeterministicDispersalKernel<IntegerRaster> deterministic_kernel;
+    bool deterministic_;
 public:
     RadialDispersalKernel(double ew_res, double ns_res,
                           DispersalKernelType dispersal_kernel,
-                          double distance_scale,
+                          double distance_scale = 1.0,
                           Direction dispersal_direction = Direction::None,
-                          double dispersal_direction_kappa = 0
+                          double dispersal_direction_kappa = 0,
+                          bool deterministic = false,
+                          const IntegerRaster& dispersers = {{0}},
+                          double dispersal_percentage = 0.99,
+                          double locator = 0.0
```

Comment:
We may want to just have this hard-coded to zero in the distribution (thoughts @wenzeslaus) and not a parameter as we always want the location to be the center of the cell, which would make it more consistent with the current implementation of the Cauchy distribution.

Reply:
I'm not sure right now what is the right place, but +1 for hardcoding, so it is always a center of the cell.

Reply:
Would you also like for me to remove the locator parameter from the deterministic kernel constructor and the CauchyDistribution pdf and icdf functions?